From 1f14ea12d232b800b43bdfdcabaff4cae7c92b3b Mon Sep 17 00:00:00 2001
From: Guo Yi <yi.guo@cavium.com>
Date: Tue, 28 May 2019 11:10:52 -0700
Subject: [PATCH 264/386] dts: cn9130-crb: change connection on eth0 to no phy
 mode

eth0 on cn9130-crb is connected directly to onboard switch instead
of a phy. Change the managed mode to "in-band-status" to enable the
port without a phy.
switch on mdio is not a phy, thus removed from dts

Change-Id: Ic18662e006bd0f638843371c5358eb75641bd673
Signed-off-by: Yi Guo <yguo@cavium.com>
Reviewed-on: https://sj1git1.cavium.com/10124
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/cn9130-crb.dtsi | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
index ff52c157ccb7..fd6a25d6b8c8 100644
--- a/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
+++ b/arch/arm64/boot/dts/marvell/cn9130-crb.dtsi
@@ -134,9 +134,6 @@
 	phy0: ethernet-phy@0 {
 		reg = <0>;
 	};
-	switch6: ethernet-phy@6 {
-		reg = <6>;
-	};
 };
 
 &cp0_xmdio {
@@ -153,8 +150,8 @@
 
 &cp0_eth0 {
 	status = "okay";
-	phy = <&switch6>;
 	phy-mode = "10gbase-kr";
+	managed = "in-band-status";
 	phys = <&cp0_comphy4 0>;
 };
 
-- 
2.17.1

